libs/librepcb/workspace/library/workspacelibrarydb.cpp

```diff
@@ -57,23 +57,25 @@ WorkspaceLibraryDb::WorkspaceLibraryDb(Workspace& ws)
   qDebug("Load workspace library database...");
 
   // open SQLite database
-  FilePath dbFilePath = ws.getLibrariesPath().getPathTo("cache.sqlite");
-  mDb.reset(new SQLiteDatabase(dbFilePath));  // can throw
+  mFilePath = ws.getLibrariesPath().getPathTo(
+      QString("cache_v%1.sqlite").arg(sCurrentDbVersion));
+  mDb.reset(new SQLiteDatabase(mFilePath));  // can throw
```

Comment:
No we don't remove old databases because we can't know if the user still wants to open the workspace with an older application version. But I think we won't introduce a new database version very often, so there won't be too many old databases. And if the user really cares about these few megabytes, he could remove them by himself.